packages/aws-cdk-lib/aws-rds/lib/aurora-cluster-instance.ts

```diff
@@ -456,7 +456,7 @@ class AuroraClusterInstance extends Resource implements IAuroraClusterInstance {
 
     const instanceParameterGroup = props.parameterGroup ?? (
       props.parameters
-        ? new ParameterGroup(props.cluster, 'InstanceParameterGroup', {
+        ? new ParameterGroup(props.cluster, `${id}InstanceParameterGroup`, {
```

Comment:
I reckon this would be better:
```suggestion
        ? new ParameterGroup(this, `InstanceParameterGroup`, {
```

Reply:
OK, I changed!